src/common/vana_time.h

```diff
@@ -2,6 +2,7 @@
 ===========================================================================
 
   Copyright (c) 2010-2015 Darkstar Dev Teams
+  Copyright (c) 2025 LandSandBoat Dev Teams
```

Comment:
You've essentially completely rewritten this, you don't need to keep around the DSP attribution unless you want to